# This fix is needed for sqlite3 to run on Solaris.  See Bug 15566164 and
# Bug 15705787 for details.
# Need to determine if these changes are still applicable to the latest version,
# and if so, then submit them upstream. 
--- a/Makefile.am.orig	2011-03-29 15:44:46.468935251 +0800
+++ b/Makefile.am	2011-03-29 15:50:12.071188848 +0800
@@ -8,6 +8,7 @@
 bin_PROGRAMS = sqlite3
 sqlite3_SOURCES = shell.c sqlite3.h
 sqlite3_LDADD = $(top_builddir)/libsqlite3.la @READLINE_LIBS@
+sqlite3_LDFLAGS = -Wl,-zrecord
 sqlite3_DEPENDENCIES = $(top_builddir)/libsqlite3.la
 
 include_HEADERS = sqlite3.h sqlite3ext.h
